### ISSUE TITLE
"Failed to create bones" error in release build

### ISSUE THREAD

**Issue by Sublustris** [defold/extension-spine#189]
Defold 1.9.5
extension-spine-3.5.3

When trying to launch windows bundle (debug), i see the following error
```
ERROR:GAMEOBJECT: The game object instance could not be created since the buffer is full (35). Increase the capacity with collection.max_instances
ERROR:SPINEEXT: Failed to create bone game object
ERROR:SPINEEXT: Failed to create bones
ERROR:SPINEEXT: Failed to create game objects for bones in spine model. Consider increasing collection max instances (collection.max_instances).
WARNING:RESOURCE: Unable to create resource: /main/main.collectionc: FORMAT_ERROR
```
When building from editor there is no errors. I've tried to increase described parameter, as follows

![изображение](https://github.com/user-attachments/assets/9e7b1623-fe67-4478-95a5-2213c570ad5f)

But nothing happens. Error is the same.

**Comment by AGulev** [comment]
Workaround: create a factory with a go in this colelction

**Comment by AGulev** [comment]
The isuue fixed in alpha, and I'll cherry-pick the fix into 1.9.5
